moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -182,8 +182,7 @@ def create_ikfast_package(args):
     user_name = getuser()
     root.append(xmlElement("maintainer", email="%s@todo.todo" % user_name, text=user_name))
     root.append(xmlElement("buildtool_depend", text="catkin"))
-    with open(pkg_xml_path, 'w') as f:
-      etree.ElementTree(root).write(f, xml_declaration=True, pretty_print=True, encoding="UTF-8")
+    etree.ElementTree(root).write(pkg_xml_path, xml_declaration=True, pretty_print=True, encoding="UTF-8")
```

Comment:
`etree` now manages opening files itself?

Reply:
Yes. It supports both file-based writing and opening files itself. However, file-based writing requires to open the file in binary mode: https://bugs.launchpad.net/lxml/+bug/1200352
I thought, just passing the file name is simpler.